app/main.py

```diff
@@ -1,13 +1,27 @@
 def format_linter_error(error: dict) -> dict:
-    # write your code here
-    pass
+    return dict(
+        line=error["line_number"],
+        column=error["column_number"],
+        message=error["text"],
+        name=error["code"],
+        source="flake8"
+    )
 
 
 def format_single_linter_file(file_path: str, errors: list) -> dict:
-    # write your code here
-    pass
+    return {
+        "errors": [format_linter_error(one) for one in errors],
+        "path": file_path,
+        "status": "failed" if errors != [] else "passed"
```

Comment:
Empty lists evaluate to False in a boolean context. So you can just check the truthiness of the list to determine if it's empty or not. If errors is empty, it evaluates to False, otherwise True.

Reply:
Ok, good advice